packages/cli/src/commands/doctor/healthchecks/reactNativeGlobal.ts

```diff
@@ -78,82 +88,17 @@ const removeNodePackage = async (packageName, packageManager, loader) => {
 export default {
   label: label,
   getDiagnostics: async () => ({
-    needsToBeFixed: checkGlobalInstall(),
+    needsToBeFixed: !!checkGlobalInstallalations(),
```

Comment:
Since I suggested to turn the response of `checkGlobalInstallalations()` in an array, you can do this:

```suggestion
    needsToBeFixed: checkGlobalInstallalations().length > 0,
```

Also, you can put this check at the top and assign it to a variable so you don't need to do it twice.